content/post/prioritization_meeting.md

```diff
@@ -0,0 +1,64 @@
+
+<!--more-->
+
+Despite those hurdles, I have come to believe that the end of sprint prioritization meeting is, save for sprint planning, the most crucial project management element of a successful sprint. In this post I will discuss some of the benefit of this meeting, including:
+
+<li>Helping to set stakeholder expectations
+<li>Increasing transparency into Data team priorities
+<li>Improving ticket/story quality
+<li>Visibility into what analysts do
+
+Hopefully by the end you will consider implementing a sprint prioritization meeting for your team.
+
+### Sprint prioritization basics
+
+The key focus areas of our scrum process are: sprint planning, daily stand ups, backlog grooming, end of sprint retro and sprint prioritization. The hour-long prioritization meeting occurs at the end of each bi-weekly sprint, is facilitated by either a scrum master or Data lead, and includes representatives of each function that Data supports. That is, if a function in the organization has work that they would like the Data team to work on in the upcoming sprint, that function would send a representative to the
+meeting. Others are welcome to sit in or dial in, but at minimum some stakeholder representation is required.
+
+The purpose of the meeting is essentially for stakeholders to work with Data to come up with a rank-ordered list of tickets for consideration in Data's next sprint. The entire group must come to a consensus around which ticket should be ranked first, second, etc... The meeting facilitator literally moves tickets up and down the board and the group decides (debates?) where a ticket should land. This way the Data team's workload is aligned with the organizations priorities without Data having to chase down stakeholders and decide what is the most important initiative this sprint. If Marketing and Product disagree on what is most important, the entire room together can weigh in with the end result being one ticket is ranked above the other. Data should opine on the technical feasibility or difficulty of executing certain tasks but in general should be unbiased participants. Data should act as tie breakers, not king makers. When prioritization works well, it is a thing of beauty for some of the following reasons.
```

Comment:
> When prioritization works well, it is a thing of beauty for some of the following reasons.

Either change the period to colon, or rewrite to something like

> I discuss the benefits of a good prioritization process below.

(Or consider striking the sentence altogether.)

Reply:
Struck